roles/sap_ha_set_hana/defaults/main.yml

```diff
@@ -1,23 +1,14 @@
 ---
-
-# az | aws | ...
-sap_ha_set_hana_type: ""
-
-# Specific variables per node - 'primary' or 'secondary'
-sap_ha_set_hana_role: ""
-
-# Part - '1' or '2'
-sap_ha_set_hana_part: ""
-
 sap_ha_set_hana_sid: "{{ sap_hana_sid }}"
 sap_ha_set_hana_instance_number: "{{ sap_hana_instance_number }}"
+sap_ha_set_hana_cluster_nodes: "{{ sap_hana_cluster_nodes }}"
 
-sap_ha_set_hana_load_balancer_ip: ""
-
-sap_ha_set_hana_site1: ""
-sap_ha_set_hana_site2: ""
+sap_ha_set_hana_vip1: "{{ sap_hana_vip1 }}"
+sap_ha_set_hana_vip2: "{{ sap_hana_vip2 }}"
 
+# az | aws | ...
```

Comment:
For readability of end users, can this line please list all types? Of refer back to a specific README link anchor for where these are listed?